src/ray/gcs/accessor.h

```diff
@@ -772,6 +772,35 @@ class PlacementGroupInfoAccessor {
   PlacementGroupInfoAccessor() = default;
 };
 
+class InternalKVAccessor {
+ public:
+  virtual ~InternalKVAccessor() = default;
+
+  virtual Status AsyncInternalKVKeys(
```

Comment:
These functions should probably have documenting comments like the other accessor APIs, even though these functions are very simple.

Reply:
Sure, I'll follow the convention.

Well, sometimes, I feel that we're adding too many not useful comments, like the comments which are not necessary, for example [here](https://github.com/ray-project/ray/blob/3869c9d342a425bc48f9c8df30ce1be02e633cda/src/ray/gcs/gcs_server/gcs_server.cc#L74).

Reply:
I completely agree about those kind of comments, although I'm a bit of a stickler about documenting abstraction interfaces, even if the semantics are pretty obvious from the function name and signature. 😛

Reply:
Yes, I'm good with API-level documentation. It's necessary.